### ISSUE TITLE
Component tag isn't recognized when importing the component from a folder above the app's root

### ISSUE THREAD

**Issue by eyalellenbogen** [angular/vscode-ng-language-service#244]
VS Code version 1.22.1
Extension version 0.1.9

I have a basic angular 5 app generated from the CLI. When I import a component that sits in a library in a folder above my app root, the language service doesn't recognize it. It adds a red underline and the tooltip says
```
[Angular]
'test-comp' is not a known element:
...
```

my folder structure is 
```
root
-- my-app
-- my-component-lib
```
and I'm importing a component from `my-component-lib` into `my-app`, registering it in the module and using it in the HTML template.

The app works fine when I run it. The component is imported and recognized properly.

**Comment by kaplan81** [comment]
Same here. I just built an Angular Component Library with the Angular CLI and everything compiles fine but I get an unfriendly red underlined component tag in the template. In other issues they point out that this might be due to the fact that I am importing the module with an absolute path instead of using a relative one but I have tried and it makes no difference. I believe it has to do with what @eyalellenbogen points out.
